bedrock/firefox/templates/firefox/all-unified.html

```diff
@@ -90,8 +90,10 @@ <h1 class="c-intro-heading">{{ ftl('firefox-all-choose-which-firefox') }}</h1>
         </div>
         <div class="c-support-links">
           <ul>
+            <li><a href="{{ url('firefox.more') }}">{{ ftl('firefox-all-learn-more-about-firefox') }}</a>
```

Comment:
I think this was part of my branch but shouldn't be. We decided against the link there in https://github.com/mozilla/bedrock/issues/9610. It think I just accidentally added this to the /more branch.